install.sh

```diff
@@ -58,19 +58,24 @@ if [[ ! -e "$INSTALL_DIR" ]] ; then
   exit 1
 fi
 
-install_tools() {
+install_symlinks() {
   info "Creating symlinks for binaries found in /native"
 
   SYMLINKS=$(cat ./src/symlinks.txt)
 
   # Note Values for ${binary} must be the full path
   for binary in $SYMLINKS; do
     if [[ ! -e $INSTALL_DIR${binary} && ! -L $INSTALL_DIR${binary} ]]; then
-      chroot $INSTALL_DIR ln -s /native${binary} ${binary}
+      mkdir -p "$(dirname "$INSTALL_DIR${binary}")"
+      ln -s /native${binary} "$INSTALL_DIR${binary}"
```

Comment:
Why drop the chroot?

Reply:
The real reason is that this script is getting called by portage as part of `src_install`. Portage script can't execute `chroot`. It didn't seem necessary to chroot just to create a symlink so I changed it. If you want I can duplicate this code for gentoo.

Reply:
Why can't portage chroot? That seems like a severe deficiency in portage.

At the very least, you should make the chroot conditional where only gentoo skips the chroot.

Reply:
It's by [design](https://devmanual.gentoo.org/general-concepts/sandbox/index.html). Portage also has packages intall into an empty `staging` directory. Then portage takes care of moving the files. So chrooting into an empty directory won't work. I unfortunalty need to [disable](https://github.com/ismell/gentoo-lx-brand-image-builder/blob/master/setup.sh#L83) most of the portage sandbox since the syscalls are not supported.

I like the idea of makign the chroot conditional. Thanks for the suggestion.

Reply:
I don't know that the chroot buys us much here anyway.  I could be convinced otherwise.  In fact in the omnios branch I have to drop the chroot because this is being installed from within a docker container and native is not there.

Reply:
Left the chroot for now.